extensions/amp-analytics/0.1/events.js

```diff
@@ -1468,6 +1472,29 @@ export class VisibilityTracker extends EventTracker {
       );
     }
 
+    const getUnlistenPromiseForSelector = (selector, selectionMethod) => {
+      return this.root
+        .getAmpElement(
```

Comment:
To confirm, `getAmpElement` doesn't support selecting mutiple elements right? Are we adding `querySelectorAll` support in a separate PR?

Reply:
Yes and yes.

See original I2I (#26823) for PR split.